CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 8.1.7 - 2020-01-29
+### Fixed
+- Add a check to see if the state is a variable
```

Comment:
Welke state? In welk bestand? In PHP is bijna alles een variabele :slightly_smiling_face:
Ik vind dit niet een heel duidelijke melding, verwerk in deze melding tenminste de `EmbedHelper` en vertel dat het over de methode `getFormState` gaat. Bijv "Fixed bug in `EmbedHelper::getFormState`..."